### DISCUSSION TITLE
Clearing Validation Indicators

### DISCUSSION THREAD

**Discussion by lempface** [canton7/Stylet#222] [General]
I have a screen using a fluentvalidator and it works really well and highlights any field that has an error. 

I have an action that creates a note in a database and ensures that the note text field is populated using Validate(). After the note is created I execute a RefreshScreen method that clears the note text field and the validation indicator is still on that field. I'd like to clear the indicator so nothing is red on the screen until the users executes the create action again and Validate() is called again.

I've tried ClearAllPropertyErrors() in inside of my if (Validate() { ... } block, but it does not work. If I put ClearAllPropertyErrors() in my RefreshScreen method and call RefreshScreen from my create note action it also does not work. But if I click the button on my UI that is bound to my RefreshScreen action then the indicators are cleared.

Any help would be appreciated!

**Comment by canton7** [comment]
Sounds odd: changing the property is enough to trigger re-validation. Could you put together a [minimal reproducible sample](https://stackoverflow.com/help/minimal-reproducible-example) which reproduces the error and which I can run, please? Perhaps using [Stylet.Samples.ModelValidation](https://github.com/canton7/Stylet/tree/master/Samples/Stylet.Samples.ModelValidation) as a starting point, if that's easier.
